CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
```

Comment:
When you complete your analysis on ways to modify font size, can you add it to shared docs? I think this is a question that can come up again (at least as long as we're using WebView).